.docker/entrypoint/lib/commands/verifier.rb

```diff
@@ -48,34 +63,29 @@ def options(ignore_urls = nil)
 
           opts = default_options
           log_level = @context.arguments.log_level
-          opts[:log_level] = log_level.to_sym unless log_level.nil?
+          opts[:log_level] = ":#{log_level}" unless log_level.nil?
           opts[:url_ignore] = ignore_urls if ignore_urls.valid_array?
 
           opts
         end
 
         def default_options
-          opts = {
+          {
             assume_extension: true,
             check_html: true,
+            check_favicon: true,
+            check_opengraph: true,
             enforce_https: true,
             only_4xx: true,
-            check_unrendered_link: true
-          }
-
-          token = @context.auth_token
-          opts[:domain_auth] = domain_auth_options(token) unless token.nil?
-          opts
-        end
-
-        def domain_auth_options(auth_token)
-          {
-            'github.com' => {
-              type: :header,
-              template: 'Bearer %token%',
-              values: {
-                token: auth_token
-              }
+            check_unrendered_link: true,
+            report_mismatched_tags: true,
+            parallel: { in_processes: Concurrent.processor_count },
+            typheous: {
+              verbose: @context.verbose?
+            },
+            verbose: @context.verbose?,
+            cache: {
+              timeframe: "1h"
```

Comment:
Codacy found an issue: [Prefer single-quoted strings when you don't need string interpolation or special symbols.](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=6155348)